serving/ingress/pkg/reconciler/ingress/resources/route_test.go

```diff
@@ -258,7 +259,7 @@ func TestMakeRoute(t *testing.T) {
 			name: "valid, passthrough by BYO cert",
 			ingress: ingress(
 				withTLS(networkingv1alpha1.IngressTLS{Hosts: []string{"custom.example.com"}, SecretName: "someSecretName"}),
-				withRules(rule(withHosts([]string{localDomain, externalDomain}))),
+				withRules(rule(withHosts([]string{"custom.example.com"}))),
```

Comment:
Serving now expects the TLS hosts to be the same as the route rule hosts to figure out what is public/cluster-local, so aligned these.